<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_Info">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>281</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_netcard">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>11</y>
       <width>63</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>网卡</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_netcard">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_IPv4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>51</y>
       <width>63</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>IPv4</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_IPv4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>191</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_MAC">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>63</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>MAC</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_MAC">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>191</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxs_can">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>281</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_scan_range">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>261</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLineEdit" name="lineEdit_IP">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>101</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_scan">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>61</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>嗅探</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_scanIP">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>63</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>嗅探IP</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_scan_catch">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>50</y>
        <width>61</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>抓包</string>
      </property>
     </widget>
    </widget>
    <widget class="QTableWidget" name="tableWidget_ip_mac">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>110</y>
       <width>251</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>471</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>411</width>
       <height>391</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_attack_srcip">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>113</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_attack_srcMAC">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>113</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_IPv4_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>63</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>源IP</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_MAC_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>63</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>源MAC</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_attack">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>110</y>
       <width>86</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>攻击</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_attack_dstip">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>113</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_attack_dstMAC">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>113</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_IPv4_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>63</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>目的IP</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_MAC_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>63</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>目的MAC</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>79</width>
       <height>30</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>定向</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>广播</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
